src/filterRows.js

```diff
@@ -56,7 +56,7 @@ function getFilterMethod(filter) {
                 .filter(cell => {
                     const hay = stringCompareValue(cell);
                     const needle = (keyword || '').toLowerCase();
-                    return !needle || hay.includes(needle);
+                    return !needle || hay.includes(needle) || hay.replace(/\,/g, '').includes(needle);
```

Comment:
This only handles case when there are `,` in the text and even so, it might not be the expected behaviour. This doesn't solve anything.